compiler/lib/js_output.ml

```diff
@@ -2101,7 +2101,8 @@ let program ?(accept_unnamed_var = false) ?(source_map = false) f p =
     let accept_unnamed_var = accept_unnamed_var
   end) in
   PP.set_needed_space_function f need_space;
-  if Config.Flag.effects () then PP.set_adjust_indentation_function f (fun n -> n mod 40);
+  if Option.is_some (Config.effects ())
```

Comment:
I would prefer an exhaustive `match .. with` here so that we reconsider the need for this every time one add a backend